modules/core/include/opencv2/core/cvstd.hpp

```diff
@@ -495,10 +495,10 @@ class CV_EXPORTS String
     explicit String();
     String(const String& str);
     String(const String& str, size_t pos, size_t len = npos);
-    String(const char* s);
-    String(const char* s, size_t n);
-    String(size_t n, char c);
-    String(const char* first, const char* last);
+    /* explicit */ String(const char* s);
```

Comment:
why did you add the explicit comment? the `const char*` converting constructor should not be explicit and the two argument constructors can not be explicit.